features/dataset-view.feature

```diff
@@ -277,6 +277,60 @@ Feature: a user visit the dataset page
 		| pre_03AUG2015_update 								| folder containing originally submitted data files, prior to update Aug 3rd 2015.	| Directory 		| 50.00 MiB 	|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/pre_03AUG2015_update |
 		| readme.txt 										|				                                                                    | Readme 			| 337 B 		|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/readme.txt |
 
+	@ok @files @javascript @pr437
+	Scenario: Files - Pagination
+		Given I am not logged in to Gigadb web site
+		And I am on "/dataset/101001"
+		And I follow "Files"
```

Comment:
Behat tests with the @javascript tag breaks at this step for me. I think this is because the chrome container needs to be built and started by adding it to the `depends_on` section for the `webapp` service in `docker-compose.yml`.